app/controllers/katello/api/rhsm/candlepin_dynflow_proxy_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Katello
+  class Api::Rhsm::CandlepinDynflowProxyController < Api::V2::ApiController
```

Comment:
I expect this action getting a completely different handling, since it won't have the payload serialized.